### PR TITLE
More versatile ToggleGroup - add NavToggleGroup - enhance Notificatio…

### PR DESCRIPTION
**Pull request by nparigi-ledger**
…n component

I added a template toggle group to use for the different toggle group (button, nav, tab) and added the navigationToggleGroup
NavigationToggleGroup used in Notification center navigation ( https://www.figma.com/file/uQYdD0E0ELDKi3skrduZpw/LLM-%2F-Notification-Center?node-id=1306%3A1293 )
It's not a breaking change

I also changed the behavior of Notification/News component. There was some fixed string, not good.
It's breaking change, however it seems that is not used yet in V3 LLM

This PR is needed for https://github.com/LedgerHQ/ledger-live-mobile/pull/2010 LL-8031 merge

### CODE REVIEW

File: packages/native/src/components/message/Notification/index.tsx [hunk 1]
```diff
@@ -68,33 +77,29 @@ export default function Notification({
...
-              color ||
-              (variant === "primary"
+              variant === "primary"
                 ? colors.palette.neutral.c00
-                : colors.palette.neutral.c80)
+                : colors.palette.neutral.c80
             }
-            mt={"2px"}
-            mb={"2px"}
+            mt={2}
```
Comment (gsoares-ledger) [inline packages/native/src/components/message/Notification/index.tsx:84]: When you input a number, it actually references the index on the "space" property of the theme, doesn't it?
So in this case you're actually setting margin top to 4px, is that intentional?

File: packages/native/src/components/Navigation/ToggleGroup/NavigationToggleGroup/index.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  text-align: center;
+  margin: auto;
+  flex: 1;
+  padding: ${(p) => p.theme.space[5]}px 0;
+  border-radius: 8px;
+  ${(p) =>
+    p.isActive &&
+    `
+    background-color: ${p.theme.colors.palette.primary.c20};
+  `}
```
Comment (gsoares-ledger) [inline packages/native/src/components/Navigation/ToggleGroup/NavigationToggleGroup/index.tsx:20]: (optional/nitpicking): you can use 
`background-color: ${(p) => p.isActive ? p.theme.colors.palette.primary.c20 : 'unset'};`
here if you'd like, might save some lines

**Review by gsoares-ledger — COMMENTED** [review]
In the React version we're calling this components "Tabs/Graph" and "Tabs/Chip". We should probably agree on some name and make them consistent.

File: packages/native/src/components/message/Notification/index.tsx (hunk 1 above)
Comment (nparigi-ledger) [reply packages/native/src/components/message/Notification/index.tsx:84]: Yes it's on purpose :ok_hand:

File: packages/native/src/components/Navigation/ToggleGroup/NavigationToggleGroup/index.tsx (hunk 2 above)
Comment (nparigi-ledger) [reply packages/native/src/components/Navigation/ToggleGroup/NavigationToggleGroup/index.tsx:20]: I will use yours, it's way better than template inside template, thank you

File: packages/native/src/components/Navigation/ToggleGroup/index.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export { default as TemplateToggleGroup } from "./TemplateToggleGroup";
```
[the remark this answers is not in the document]
Comment (nparigi-ledger) [reply packages/native/src/components/Navigation/ToggleGroup/index.ts:1]: I wasn't sure if it could have been useful yeah.

**Comment by elbywan** [comment]
@nparigi-ledger Can you revert your merge and rebase instead? Thanks 🙏

**Comment by nparigi-ledger** [comment]
@elbywan i just pushed some changes after the merge, didn't see your message sorry, can you let it pass one last time ? :pray:

**Comment by elbywan** [comment]
> i just pushed some changes after the merge, didn't see your message sorry, can you let it pass one last time ? 🙏

@nparigi-ledger Haha ok last one then 👌

File: packages/native/src/components/Navigation/ToggleGroup/index.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (nparigi-ledger) [reply packages/native/src/components/Navigation/ToggleGroup/index.ts]: Done

**Comment by nparigi-ledger** [comment]
@gsoares-ledger Renamed these components Tabs/Graph Tabs/Chip following your advice on keeping the same name as desktop (good thing to do indeed)

**Review by gsoares-ledger — APPROVED** [review]

File: packages/native/src/components/Navigation/ToggleGroup/NavigationToggleGroup/index.tsx (hunk 2 above)
Comment (nparigi-ledger) [reply packages/native/src/components/Navigation/ToggleGroup/NavigationToggleGroup/index.tsx]: Little note about this change that i learnt and that could be useful for you too @gsoares-ledger  :smile: 
backgroundColor can't be 'unset' in react-native

**Pull request merged**